dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -49,8 +49,11 @@
 
 -# Blockly script dependencies
 - if use_blockly
-  %script{src: webpack_asset_path('js/blockly.js')}
-  %script{src: webpack_asset_path("js/#{js_locale}/blockly_locale.js")}
+  - if use_google_blockly
+    %script{src: webpack_asset_path('js/googleblockly.js')}
+  - else
+    %script{src: webpack_asset_path('js/blockly.js')}
+    %script{src: webpack_asset_path("js/#{js_locale}/blockly_locale.js")}
```

Comment:
yes! 👍  this looks like it will keep google blockly out of our regular users' browsers.